src/constants.ts

```diff
@@ -86,3 +86,59 @@ export const MARKET_DEPTH_END_PRICE_SLIPPAGE_PERC = 20;
 
 // Logging
 export const NUMBER_SOURCES_PER_LOG_LINE = 12;
+
+// ABI for balance checker
+export const BALANCE_CHECKER_ABI = [
+    {
+        constant: true,
+        inputs: [
+            { name: 'user', type: 'address' },
+            { name: 'token', type: 'address' },
+        ],
+        name: 'tokenBalance',
```

Comment:
do we need `tokenBalance` or is it just an internal function?

Reply:
It's callable. I'm just using the entire contract ABI here